docs/_data/table/cn/user-configuration.yml

```diff
@@ -25,6 +25,8 @@ alluxio.user.file.readtype.default:
 alluxio.user.file.writetype.default:
   创建Alluxio文件时的默认写类型。可选值为`MUST_CACHE` (数据仅仅存储在Alluxio中，并且必须存储在其中),
   `CACHE_THROUGH` (尽量缓冲数据，同时同步写入到底层文件系统), `THROUGH` (不缓冲数据，同步写入到底层文件系统)。
+alluxio.user.file.cache.incomplete.block:
+  当读取类型是 CACHE 或者 CACHE_PROMOTE 的时候, 如果设置这个变量为真, 没有完全读取的数据块也会被存到Alluxio内.
```

Comment:
NIT: should we quote `CACHE` and `CACHE_PROMOTE` like <pre>\`CACHE\`</pre>  here?